README.md

```diff
@@ -3,18 +3,18 @@
 [![GitHub release](https://img.shields.io/badge/release-download-orange.svg)](https://github.org/apache/rocketmqreleases)
 [![License](https://img.shields.io/badge/license-Apache%202-4EB1BA.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
 
-**[Apache RocketMQ](https://rocketmq.incubator.apache.org) is a low latency, reliable, scalable, easy to use message oriented middleware born from alibaba massive messaging business.**
+**[Apache RocketMQ](https://rocketmq.incubator.apache.org) is an open source distributed messaging and streaming data platform with low latency, high reliability, trillion-level capacity, well-structed concurrency and flexible scalability.**
 
-It offers a variety of features as follows:
+It offers a variety of features:
 
-* Pub/Sub and P2P messaging model
+* Pub/Sub messaging model
 * Reliable FIFO and strict sequential messaging in the same queue
-* Long pull queue model,also support push consumption style
-* Million message accumulation ability in single queue
-* Over a variety of messaging protocols.such as JMS,MQTT etc.
-* Distributed high available deploy architecture, meets at least once message delivery semantics
-* Docker images for isolated testing and cloud Isolated clusters
-* Feature-rich administrative dashboard for configuration,metrics and monitoring
+* Long pull queue model and push consumption style
+* Million-level message accumulation capacity in single queue
+* Useful messaging protocols like [JMS](http://docs.oracle.com/javaee/6/tutorial/doc/bncdq.html)
+* Distributed highly available deployment architecture, meeting at least one message delivery semantic
```

Comment:
`At-most-once, At-least-once, Exactly-once`, no need to replace `once` with `one`.